### PR TITLE
changing the syntax to match the standard one

### PR DESCRIPTION
**Pull request by ahaffemayer**
i broke my API so i didn't run the soft but it builds. I'll repare it tomorrow

### CODE REVIEW

File: include/panda_torque_mpc/crocoddyl_reaching.h [hunk 1]
```diff
@@ -64,8 +64,8 @@ namespace panda_torque_mpc
             // dummy constructor necessary to use this class as a member variable directly
         }
 
-        CrocoddylReaching(pin::Model _model_pin, const boost::shared_ptr<pin::GeometryModel>& _collision_model ,CrocoddylConfig _config);
-        // CrocoddylReaching(pin::Model _model_pin, CrocoddylConfig _config);
+        CrocoddylReaching(pin::Model model_pin_, const boost::shared_ptr<pin::GeometryModel>& collision_model_ ,CrocoddylConfig _config);
+        // CrocoddylReaching(pin::Model model_pin_, CrocoddylConfig _config);
```
Comment (Kotochleb) [inline include/panda_torque_mpc/crocoddyl_reaching.h:68]:
```suggestion
        CrocoddylReaching(const pin::Model& model_pin, const boost::shared_ptr<pin::GeometryModel>& collision_model ,const CrocoddylConfig& config);
```
Underscore means internal variable of a class. Parameters of a function should be passed with the naming convention, like with regular variables. The underscore after the variable is used to distinguish it as a class member. The same logic is in python with a leading underscore, being said "protected" variable or function.

File: src/crocoddyl_motion_server_node.cpp [hunk 2]
```diff
@@ -125,8 +125,8 @@ namespace panda_torque_mpc
             std::string urdf_path = ros::package::getPath("panda_torque_mpc") + "/urdf/robot.urdf";
 
             // Building the GeometryModel
-            auto collision_model = boost::make_shared<pinocchio::GeometryModel>();
-            collision_model = loadPandaGeometryModel(model_pin_);
+            auto collision_model_ = boost::make_shared<pinocchio::GeometryModel>();
+            collision_model_ = loadPandaGeometryModel(model_pin_);
```
Comment (Kotochleb) [inline src/crocoddyl_motion_server_node.cpp:129]:
```suggestion
            auto collision_model = loadPandaGeometryModel(model_pin_);
```
Same as before. This is not a class member. Plus, this object is first initialized in your case with empty object to later be overwritten. The same result will be with this code.

File: src/crocoddyl_motion_server_node.cpp [hunk 3]
```diff
@@ -138,26 +138,26 @@ namespace panda_torque_mpc
...
 
-            collision_model->addCollisionPair(pinocchio::CollisionPair(collision_model->getGeometryId("obstacle"),
-                collision_model->getGeometryId("panda_link7_sc_1")));
+            collision_model_->addCollisionPair(pinocchio::CollisionPair(collision_model_->getGeometryId("obstacle"),
+                collision_model_->getGeometryId("panda_link7_sc_1")));
 
-            collision_model->addCollisionPair(pinocchio::CollisionPair(collision_model->getGeometryId("obstacle"),
-                collision_model->getGeometryId("panda_link7_sc_4")));
+            collision_model_->addCollisionPair(pinocchio::CollisionPair(collision_model_->getGeometryId("obstacle"),
+                collision_model_->getGeometryId("panda_link7_sc_4")));
```
Comment (Kotochleb) [inline src/crocoddyl_motion_server_node.cpp:160]: You should revert this change

File: src/crocoddyl_reaching.cpp [hunk 4]
```diff
@@ -37,15 +37,15 @@
 
 namespace panda_torque_mpc
 {
-    CrocoddylReaching::CrocoddylReaching(pin::Model _model_pin, const boost::shared_ptr<pin::GeometryModel>& _collision_model ,
+    CrocoddylReaching::CrocoddylReaching(pin::Model model_pin_, const boost::shared_ptr<pin::GeometryModel>& collision_model_ ,
         CrocoddylConfig _config) :
     config_(_config)
```
Comment (Kotochleb) [inline src/crocoddyl_reaching.cpp:42]:
```suggestion
    CrocoddylReaching::CrocoddylReaching(const pin::Model& model_pin, const boost::shared_ptr<pin::GeometryModel>& collision_model ,
        const CrocoddylConfig& config) :
    config_(config)
```

File: src/crocoddyl_reaching.cpp [hunk 5]
```diff
@@ -37,15 +37,15 @@
...
     {
 
-        auto end_effector_frame_id = _model_pin.getFrameId(_config.ee_frame_name);
+        auto end_effector_frame_id = model_pin_.getFrameId(_config.ee_frame_name);
```
Comment (Kotochleb) [inline src/crocoddyl_reaching.cpp:45]:
```suggestion
        const std::size_t end_effector_frame_id = model_pin.getFrameId(_config.ee_frame_name);
```
This value is considered immutable later on in the code, so it can be explicitly immutable

File: src/crocoddyl_reaching.cpp [hunk 6]
```diff
@@ -37,15 +37,15 @@
...
 
         std::cout << "Creating state, actuation and IAMs... " << std::endl;
-        auto state = boost::make_shared<crocoddyl::StateMultibody>(boost::make_shared<pinocchio::Model>(_model_pin));
+        auto state = boost::make_shared<crocoddyl::StateMultibody>(boost::make_shared<pinocchio::Model>(model_pin_));
```
Comment (Kotochleb) [inline src/crocoddyl_reaching.cpp:48]:
```suggestion
        auto state = boost::make_shared<crocoddyl::StateMultibody>(boost::make_shared<pinocchio::Model>(model_pin));
```

File: src/crocoddyl_reaching.cpp [hunk 7]
```diff
@@ -75,11 +75,11 @@ namespace panda_torque_mpc
         lower_bound << 1e-1;
         upper_bound << std::numeric_limits<double>::infinity();
 
-        for (int col_idx = 0; col_idx < _collision_model->collisionPairs.size(); col_idx++)
+        for (int col_idx = 0; col_idx < collision_model_->collisionPairs.size(); col_idx++)
```
Comment (Kotochleb) [inline src/crocoddyl_reaching.cpp:78]:
```suggestion
        for (int col_idx = 0; col_idx < collision_model->collisionPairs.size(); col_idx++)
```

File: src/crocoddyl_reaching.cpp [hunk 8]
```diff
@@ -75,11 +75,11 @@ namespace panda_torque_mpc
...
         {
 
             auto obstacle_distance_residual = boost::make_shared<colmpc::ResidualDistanceCollision>
-                (colmpc::ResidualDistanceCollision(state, 7, _collision_model, col_idx));
+                (colmpc::ResidualDistanceCollision(state, 7, collision_model_, col_idx));
```
Comment (Kotochleb) [inline src/crocoddyl_reaching.cpp:82]:
```suggestion
                (colmpc::ResidualDistanceCollision(state, 7, collision_model, col_idx));
```

File: src/crocoddyl_motion_server_node.cpp [hunk 9]
```diff
@@ -194,7 +194,7 @@ namespace panda_torque_mpc
             config_croco_.armature = Eigen::Map<Eigen::Matrix<double, 7, 1>>(armature.data());
 
             // croco_reaching_ = CrocoddylReaching(model_pin_ ,config_croco_);
-            croco_reaching_ = CrocoddylReaching(model_pin_, collision_model ,config_croco_);
+            croco_reaching_ = CrocoddylReaching(model_pin_, collision_model_ ,config_croco_);
```
Comment (Kotochleb) [inline src/crocoddyl_motion_server_node.cpp:197]:
```suggestion
            croco_reaching_ = CrocoddylReaching(model_pin_, collision_model, config_croco_);
```

**Review by Kotochleb — CHANGES_REQUESTED** [review]

**Comment by ahaffemayer** [comment]
OK let's close this PR, it doesn't bring anything to the table.
